### PR TITLE
tutorials/lodestar-A

### PR DESCRIPTION
**Pull request by Pwhsky**
updated lodestar tutorial A

### CODE REVIEW

**Comment by edudc** [comment]
There's a small typo under **4. Training LodeSTAR** that says "Initialie". Otherwise lgm.

**Comment by foca13** [comment]
Suggestions

- Specify that this example contains 10 images:
In mardown cell
1. Loading the Images of Quantum Dots
The data is found in the `tutorials/2-examples/assets` directory as an array with the format (time, height, width). This example contains 10 imatges
Then
images = np.load("./assets/image_quantum_dots.npy")
images.shape

- Plot the final predictions in a single 2x5 plot, something like:
fig, axes = plt.subplots(2, 5, figsize=(25, 8))

for i, (frame, detections) in enumerate(zip(images_tensor, all_detections)):
    ax = axes[i // 5, i % 5]
    img = frame.squeeze(0).detach().cpu()
    ax.imshow(img, aspect='auto')
    ax.scatter(
        detections[:, 1], detections[:, 0], marker="+", c="r", alpha=0.5
    )
    ax.set_xlim(0, img.shape[1])
    ax.set_ylim(img.shape[0], 0)  # inverted y, since image origin is top-left
    ax.axis("off")
    ax.set_title(f"Frame {i}")

plt.show()
